pkg/syncer/syncer.go

```diff
@@ -211,6 +237,39 @@ func StartSyncer(ctx context.Context, cfg *SyncerConfig, numSyncerThreads int, i
 		return err
 	}
 
+	ddsifForUpstreamSyncer, err := ddsif.NewDiscoveringDynamicSharedInformerFactory(upstreamSyncerClusterClient, nil, nil, resourceController, cache.Indexers{})
+	if err != nil {
+		return err
+	}
+
+	ddsifForUpstreamUpsyncer, err := ddsif.NewDiscoveringDynamicSharedInformerFactory(upstreamUpsyncerClusterClient, nil, nil,
+		&FilteredGVRSource{
+			resourceController,
+			func(gvr schema.GroupVersionResource) bool {
+				if gvr.Resource == "persistentvolumes" && gvr.Group == "" {
```

Comment:
`return gvr != corev1.Resource("persistentvolumes")`

Reply:
In fact I also added `endpoints` and `pods` in preparation to what will happen next. So it seems it makes sense to test the group separately from the resource names.